### ISSUE TITLE
[AVM Module Issue]: avm.res.container-service.managed-cluster

### ISSUE THREAD

**Issue by zensqlmonitor** [Azure/bicep-registry-modules#3819]
### Check for previous/existing GitHub issues

- [x] I have checked for previous/existing GitHub issues

### Issue Type?

Bug

### Module Name

avm/res/container-service/managed-cluster

### (Optional) Module Version

0.5.1

### Description

When deploying one or mutiple agent pool.s with osSku equals to AzureLinux, Ubuntu is deployed instead.

Deployment:
![Image](https://github.com/user-attachments/assets/761e241f-8877-4d73-975f-05db4ebcf054)

Result:
![Image](https://github.com/user-attachments/assets/2264665f-89ba-4ab3-a925-f5751b952e9b)

**Comment by zensqlmonitor** [comment]
🤔

**Comment by AlexanderSehr** [comment]
Hey @JPEasier / @ilhaan, 
if you have a moment, please triage this issue :)

**Comment by JPEasier** [comment]
Hi @zensqlmonitor,
I will take a look tomorrow and get back to you.

Thanks you.

**Comment by zensqlmonitor** [comment]
@KLuuKer @JPEasier Indeed, I just saw that the _osSku_ property is missing for the _primaryAgentPoolProfiles_.

**Comment by JPEasier** [comment]
Pull request is under review
